lib/chat_api/conversations/helpers.ex

```diff
@@ -0,0 +1,59 @@
+defmodule ChatApi.Conversations.Helpers do
+  @moduledoc """
+  Helper methods for Conversations context.
+  """
+
+  alias ChatApi.Slack
+
+  @spec send_conversation_state_update(Conversation.t(), map()) ::
+          {:ok, String.t()} | {:error, String.t()}
+  def send_conversation_state_update(conversation, state) do
+    conversation_state_message = get_conversation_state_message(state)
+
+    if conversation_state_message do
+      Slack.send_conversation_message_alert(
+        conversation.id,
+        conversation_state_message,
+        type: :conversation_update
+      )
+
+      {:ok, conversation_state_message}
+    else
+      {:error, "state_invalid"}
+    end
+  end
+
+  @spec send_multiple_archived_updates([Conversation.t()]) :: list()
+  def send_multiple_archived_updates(conversations) do
+    state = %{"status" => "archived"}
+
+    Enum.map(conversations, fn conversation ->
+      send_conversation_state_update(conversation, state)
+    end)
+  end
+
+  defp get_conversation_state_message(state) do
+    case state do
+      %{"status" => "open"} ->
+        ":outbox_tray: This conversation has been reopened."
+
+      %{"status" => "closed"} ->
+        ":white_check_mark: This conversation has been closed."
+
+      %{"status" => "archived"} ->
```

Comment:
hmm i wonder if we should use a different term here since "archived" and "deleted" aren't valid statuses (like "open" and "closed")... maybe `"state" => "archived"` instead? just trying to avoid confusion :P

Reply:
ah that's a good point!